scripts/Engineering/EnggUtils.py

```diff
@@ -6,42 +6,143 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 from os import path
 from mantid.api import *
-from mantid.kernel import IntArrayProperty, UnitConversion, DeltaEModeType
+from mantid.kernel import IntArrayProperty, UnitConversion, DeltaEModeType, logger
 import mantid.simpleapi as mantid
 from mantid.simpleapi import AnalysisDataService as ADS
 from matplotlib import gridspec
-
-ENGINX_BANKS = ['', 'North', 'South', 'Both: North, South', '1', '2']
```

Comment:
I see in the diff that you've removed this ENGINX_BANKS constant from here and inserted it separately into the EnggCalibrate\EnggCalibrateFull\EnggFocus units. What's the reason for that?

Reply:
I'd removed it because it's only used in now deprecated algorithms, but when their tests failed I just added the constant into their individual files. Thinking about it it probably makes more sense to just keep it

Reply:
Thanks. The change you've pushed looks fine. Please can you add a release note and then I think we're done with this one